cmds/unassign.go

```diff
@@ -0,0 +1,94 @@
+package cmds
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/Betterment/testtrack-cli/fakeassignments"
+	"github.com/Betterment/testtrack-cli/schema"
+	"github.com/Betterment/testtrack-cli/validations"
+	"github.com/spf13/cobra"
+)
+
+var unassignDoc = `
+Removes split assignment from all visitors in the testtrack fake server.
+
+This command can be used to remove the assignment for a specific split
+or to remove all split assignments.
+
+Example:
+
+testtrack unassign my_fancy_experiment
+testtrack unassign --all
+`
+
+var unassignVariant string
+var unassignAll bool
+
+func init() {
+	unassignCmd.Flags().BoolVar(&noPrefix, "no-prefix", false, "Don't prefix split with app_name (supports legacy splits)")
+	unassignCmd.Flags().BoolVar(&unassignAll, "all", false, "Unassign all splits")
+	rootCmd.AddCommand(unassignCmd)
+}
+
+var unassignCmd = &cobra.Command{
+	Use:   "unassign [split_name]",
+	Short: "Remove a split assignment",
+	Long:  unassignDoc,
+	Args:  cobra.RangeArgs(0, 1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		name := ""
+		if len(args) == 1 {
+			name = args[0]
+		}
+		return unassign(name, unassignAll)
+	},
+}
+
+func unassign(name string, all bool) error {
+	if all && len(name) > 0 {
```

Comment:
This could be != instead of && and that maybe gets you what you want?

Reply:
```go
nameProvided := len(name) > 0
if all == nameProvided {
  return errors.New("split_name and --all are mutually exclusive but one is required")
}
```

Reply:
oh yea, that's good.